docs/general/run-a-node/node-operator-overview.mdx

```diff
@@ -3,9 +3,62 @@ import {findSidebarItem} from '@site/src/sidebarUtils';
 
 # Overview
 
-[Oasis Network](../oasis-network/overview.md)'s Consensus Layer is a decentralised set of validator nodes that maintain a proof-of-stake blockchain. The nodes are operated by independent node operators.
+The following documentation is intended to assist any individual or 
+organisation in participating in the Oasis network as a node operator. To join 
+the network we recommend you first try running nodes on the Testnet. The 
+Testnet is a playground, where you can learn and experiment without the risk 
+of loosing real tokens. 
 
-This documentation is intended to assist any individual or organization in participating in the network as a node operator.
+[Oasis Network] consists of the Consensus Layer and ParaTimes. Consensus and 
+ParaTime nodes can be operated by anyone.
+
+:::info
+
+Consensus Layer is a decentralised set of 120 validator nodes that are a 
+backbone of the Oasis Network. The current validator set size is determined by 
+governance - the network started with 80 nodes in the validator set in 2020 and 
+has expanded to 120 nodes over the past few network upgrades. Current node 
+operators can be seen on a block explorer such as [Oasis Scan].
+
+:::
+
+:::info
+
+Operating a ParaTime Node on the Mainnet requires the participation of node 
+operators who have the validator node in the active validator set. ParaTimes 
+have their own reward system, participation requirements and structure. As a 
+node operator you can participate in any number of ParaTimes.
+
+:::
+
+If you have any questions about running a node you can find us on [Discord].
+
+[Oasis Network]: ../oasis-network/overview.md
+[Oasis Scan]: https://www.oasisscan.com/validators
+[Discord]: https://discord.gg/RwNTK8t
+
+## Set Up Your Node overview
+
+To run a **Validator node**, make sure your system meets the [Hardware] and the 
+[System] prerequisites and has [Oasis Node] installed.
+
+Then proceed by following the [Run a Validator Node] guide to:
+
+* Create your entity.
+* Initialize and configure your node.
+* Put enough stake in your escrow account.
+* Register your entity on the network.
+
+To run a **ParaTime node** make sure to first set up a Validator node. Then set 
+up a [Trusted Execution Environment (TEE)]. After that proceed to the [Run a 
+ParaTime node].
+
```

Comment:
Here, I would mention the third option - client nodes. Add links where appropriate.

```suggestion

If you are building a service on top of the Oasis network, we suggest
that you set up a **non-validator node** and (optionally) a **ParaTime client
node**, so your service do not depend on third party endpoints which are
often behind a traffic limiter.
```